reemgawad/theatr
# Clickable photo popup activity

**What changed:**
- Created the Clickable Photo with Popup activity (added to seeds + attached photos, created partial)
- Modified the "completed" action in the BadgesController so that if a student is logged in, it redirects them to their profile page where they can see that the activity is completed.

**How to test:**
- Sign in as teacher, go to settings and add 'Clickable Photos with Popups' to classroom
- Sign in as student and go to that activity, click each photo - a modal should pop up with the actor's description
- Click complete activity button, should redirect you to your profile page where the activity is marked as completed.
